src/client/main.ts

```diff
@@ -158,6 +158,9 @@ const contextListeners: ContextListener[] = [];
  * ```
  * @param name The [[AppName]] to launch.
  * @param context A context to pass to the app post-launch.
+ * @throws [[FDC3Error]] with an [[ApplicationError]] code.
+ * @throws If `context` is passed, [[FDC3Error]] with a [[SendContextError]] code.
```

Comment:
Does this line-up better in the docs if it's flipped round?

```
@throws [[FDC3Error]] with a [[SendContextError]] code, if `context` is passed.
```

Reply:
I think visually that's better, but as a sentence, it's more natural to put that first. It also gets trickier with the `TypeErrors` - since that isn't our class to document, I've put more in the `@throws` line, and I'm following the pattern

If [circumstances this error is relevant] [error type], if [specific cause of error if not documented in linked error class]